### PR TITLE
✨ feat(tools/verify.ts): add verify script

### PR DESCRIPTION
**Pull request by zh19910202**

### CODE REVIEW

**Comment by zh19910202** [comment]
The deployment information left before cannot pass the validation script. After redeploying, validation works fine. Considering that the new version of the contract has not been enabled in the rinkeby environment, the contract address information of this deployment will be used in the future.

File: tools/verify.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import env from 'hardhat';
+import path from 'path';
+import fs from 'fs-extra';
+
+const list = [
+  '/Users/snow/contracts/deployments/rinkeby/P12Token.json',
+  '/Users/snow/contracts/deployments/rinkeby/UniswapV2Factory.json',
+  '/Users/snow/contracts/deployments/rinkeby/UniswapV2Router02.json',
+  '/Users/snow/contracts/deployments/rinkeby/WETH9.json',
```
Comment (Jerrywang959) [inline tools/verify.ts:9]: use relative path instead

File: tools/verify.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        }
+      }
+    }
+  });
+}
+
+function travel(dir: string, callback: (arg0: string) => void) {
+  fs.readdir(dir, (err, files) => {
+    if (err) {
+      console.log(err);
```
Comment (Jerrywang959) [inline tools/verify.ts:40]: console.error or exit?

**Review by Jerrywang959 — APPROVED** [review]

**Review by Jerrywang959 — CHANGES_REQUESTED** [review]
approved by mistake

**Review by Jerrywang959 — APPROVED** [review]

**Pull request merged**